app/Notifications/BillHasProgressed.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Notifications;
+
+use App\Models\Group;
+use App\Models\LegiScan\Bill;
+use App\Models\LegiScan\BillHistory;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+
+class BillHasProgressed extends Notification
+{
+    use Queueable;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @return void
+     */
+    public function __construct(public BillHistory $bill_history) {}
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param  mixed  $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        return ['mail'];
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     *
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
```

Comment:
if `$notifiable` is a User, just say `$user`. This is our method now.

Reply:
Done!